src/Illuminate/Cache/Repository.php

```diff
@@ -490,7 +490,7 @@ protected function getMinutes($duration)
             $duration = Carbon::now()->diffInSeconds(Carbon::instance($duration), false) / 60;
         }
 
-        return $duration > 0 ? $duration : null;
+        return (int) ($duration * 60) > 0 ? $duration : null;
     }
```

Comment:
Oh, wait, I missread this. I read it as being:

```php
(int) (($duration * 60) > 0 ? $duration : null)
```

But, of course, PHP's precedence doesn't work like that.